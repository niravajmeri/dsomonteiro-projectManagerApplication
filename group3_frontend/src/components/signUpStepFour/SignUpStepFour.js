import React, { Component } from "react";
import { NavLink } from "react-router-dom";

class SignUpStepFour extends Component {
    state = {};
    render() {
        return (
            <div className="stepFourConfirmation">
                <div className="sucessRegistrationText">
                    Registration completed successfully!
                </div>
                <div className="LogIn">
                    <NavLink className="btn btn-primary btn-lg" to="/login">
                        Log In
                    </NavLink>
                </div>
            </div>
        );
    }
}

export default SignUpStepFour;
