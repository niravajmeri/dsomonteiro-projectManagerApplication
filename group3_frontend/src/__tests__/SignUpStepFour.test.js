import React from 'react';
import { shallow } from 'enzyme';
import SignUpStepFour from '../components/signUpStepFour/SignUpStepFour';

describe('SignUpStepFour', () => {
    it('should render correctly', () => {
        const component = shallow(<SignUpStepFour />);
        expect(component).toMatchSnapshot();
    });
});
