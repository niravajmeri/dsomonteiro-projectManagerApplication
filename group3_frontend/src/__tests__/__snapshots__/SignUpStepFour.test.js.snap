// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SignUpStepFour should render correctly 1`] = `
ShallowWrapper {
  "length": 1,
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <SignUpStepFour />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": Array [
        <div
          className="sucessRegistrationText"
>
          Registration completed successfully!
</div>,
        <div
          className="LogIn"
>
          <NavLink
                    activeClassName="active"
                    ariaCurrent="true"
                    className="btn btn-primary btn-lg"
                    to="/login"
          >
                    Log In
          </NavLink>
</div>,
      ],
      "className": "stepFourConfirmation",
    },
    "ref": null,
    "rendered": Array [
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": "Registration completed successfully!",
          "className": "sucessRegistrationText",
        },
        "ref": null,
        "rendered": "Registration completed successfully!",
        "type": "div",
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": <NavLink
            activeClassName="active"
            ariaCurrent="true"
            className="btn btn-primary btn-lg"
            to="/login"
>
            Log In
</NavLink>,
          "className": "LogIn",
        },
        "ref": null,
        "rendered": Object {
          "instance": null,
          "key": undefined,
          "nodeType": "function",
          "props": Object {
            "activeClassName": "active",
            "ariaCurrent": "true",
            "children": "Log In",
            "className": "btn btn-primary btn-lg",
            "to": "/login",
          },
          "ref": null,
          "rendered": "Log In",
          "type": [Function],
        },
        "type": "div",
      },
    ],
    "type": "div",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": Array [
          <div
            className="sucessRegistrationText"
>
            Registration completed successfully!
</div>,
          <div
            className="LogIn"
>
            <NavLink
                        activeClassName="active"
                        ariaCurrent="true"
                        className="btn btn-primary btn-lg"
                        to="/login"
            >
                        Log In
            </NavLink>
</div>,
        ],
        "className": "stepFourConfirmation",
      },
      "ref": null,
      "rendered": Array [
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": "Registration completed successfully!",
            "className": "sucessRegistrationText",
          },
          "ref": null,
          "rendered": "Registration completed successfully!",
          "type": "div",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": <NavLink
              activeClassName="active"
              ariaCurrent="true"
              className="btn btn-primary btn-lg"
              to="/login"
>
              Log In
</NavLink>,
            "className": "LogIn",
          },
          "ref": null,
          "rendered": Object {
            "instance": null,
            "key": undefined,
            "nodeType": "function",
            "props": Object {
              "activeClassName": "active",
              "ariaCurrent": "true",
              "children": "Log In",
              "className": "btn btn-primary btn-lg",
              "to": "/login",
            },
            "ref": null,
            "rendered": "Log In",
            "type": [Function],
          },
          "type": "div",
        },
      ],
      "type": "div",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
      },
    },
  },
}
`;
